projects/storefrontlib/src/shared/components/checktable/checktable.component.ts

```diff
@@ -0,0 +1,45 @@
+import { Component, Input, Output } from '@angular/core';
+import { EventEmitter } from '@angular/core';
+
+@Component({
+  selector: 'cx-checktable',
+  templateUrl: './checktable.component.html',
+})
+export class ChecktableComponent {
+  @Input()
+  columns: Array<{
+    key: string;
+    value: string;
+    cxRoute?: string;
+    checked?: boolean;
+  }>;
+
+  @Input()
+  tableData: Array<any>;
+
+  @Output() toggle = new EventEmitter<any[]>();
+
+  selectedValues: Array<any>;
+
+  onToggle(event, row) {
+    // console.log(event.target.value)
+    if (event.target.value = 'on') {
```

Comment:
why do you set that 'on' string to target.value? I think we need emit value on change value always after change `this.toggle.emit(event.target.value, row);`